docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -66,16 +69,34 @@ To generate the index just add the following to your `pom.xml`:
   </plugins>
 </build>
 ----
+****
 
-If you are using gradle, you can apply the following plugin to your `build.gradle`:
+[role="secondary asciidoc-tabs-sync-gradle-groovy"]
+.Gradle (Groovy DSL)
+****
 
-[source,groovy]
+[source,groovy,subs=attributes+]
 ----
 plugins {
-    id 'org.kordamp.gradle.jandex' version '0.11.0'
+    id 'org.kordamp.gradle.jandex' version '0.13.0'
```

Comment:
Could we make it an attribute here: https://github.com/quarkusio/quarkus/blob/main/docs/pom.xml#L2857 ?

Reply:
Right, definitely!